docs/services/sdk/overview.md

```diff
@@ -42,14 +84,14 @@ export class GreeterService implements Greeter {
 
     async countGreetings(request: GreetRequest): Promise<GreetResponse> {
         // Retrieving the Restate context
-        const restateContext = restate.useContext(this);
+        const ctx = restate.useContext(this);
 
-        // State management
-        let seen = (await restateContext.get<number>("seen")) || 0;
+        // Retrieve state; number of times this name was seen
+        let seen = (await ctx.get<number>("seen")) || 0;
         seen += 1;
-        await restateContext.set("seen", seen);
+        // Set the incremented counter as the new state
+        await ctx.set("seen", seen);
```

Comment:
Is the formatting of the imports and the methods on `restate` below correct? It looks as if some indentation is missing.